scripts/ejpcsv2json.py

```diff
@@ -30,6 +30,13 @@ def first(lst):
     except IndexError:
         return None
 
+def empty_fields_are_null(row):
+    "mutator"
+    for key, val in row.items():
+        if isinstance(val, str) and str(val).strip() == "":
```

Comment:
Same thought on mutator here (see other PR).

The strip wasn't part of the original idea. But I can't really think of useful cases where just white spaces are desirable.

@giorgiosironi @GiancarloFusiello

Reply:
I think it's good practice to have function names that describe what they do. In this case, the function replaces empty strings with `None` values so, a name like `replace_emtpy_strings_with_none_values` might be a more descriptive name which suggests side effects. Another approach is to have a series of Cleaning functions that do not have side effects like so:
```python
def is_empty_string(value):
    if not isinstance(value, str):
        raise TypeError
    return value.strip() == ''

def clean_empty_strings(row):
    return {key: value if is_empty_string(value) else None for key, value}

def clean(row):
    return clean_empty_strings(row)
```
Each one of these functions can be tested independently and more cleaning operations can be added to the main clean function. I hope this helps.